tests/subsys/rtio/rtio_api/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_minimum_required(VERSION 3.20.0)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(rtio_api_test)
 
-FILE(GLOB app_sources src/*.c)
+FILE(GLOB app_sources src/main.c)
```

Comment:
glob not needed anymore?

Reply:
Ah yes, thanks, I was trying to get rid of the glob